acquisitions/src/main/resources/thunderjet/mod-orders/features/open-order-instance-link.feature

```diff
@@ -162,6 +162,7 @@ Feature: Check opening an order links to the right instance based on the identif
     When method POST
     Then status 201
 
+    * call pause 32000
```

Comment:
We need these pause for 2 places, not for one, correct?

Reply:
And please describe why we need this pause in comments, I mean explain that we have cache with expiration 30 seconds, and for what entities this cache  applied, it will help a lot to understand and not forget about this pause

Reply:
we are updating configs at 2 places, so yes.

Reply:
> And please describe why we need this pause in comments, I mean explain that we have cache with expiration 30 seconds, and for what entities this cache applied, it will help a lot to understand and not forget about this pause

done.